pkgs/by-name/i3/i3pystatus/package.nix

```diff
@@ -0,0 +1,96 @@
+{
+  lib,
+  fetchFromGitHub,
+  libpulseaudio,
+  libnotify,
+  gobject-introspection,
+  python3Packages,
+  unstableGitUpdater,
+  fetchpatch,
+  extraLibs ? [ ],
+}:
+
+python3Packages.buildPythonApplication rec {
+  # i3pystatus moved to rolling release:
+  # https://github.com/enkore/i3pystatus/issues/584
+  version = "3.35-unstable-2024-06-13";
+  pname = "i3pystatus";
+
+  src = fetchFromGitHub {
+    owner = "enkore";
+    repo = "i3pystatus";
+    rev = "f3c539ad78ad1c54fc36e8439bf3905a784ccb34";
+    sha256 = "3AGREY+elHQk8kaoFp8AHEzk2jNC/ICGYPh2hXo2G/w=";
+  };
+
+  patches = [
+    # absolutifies the path to the test data in buds test so it can be run from anywhere
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/enkore/i3pystatus/pull/869.patch";
```

Comment:
This is not a stable path, can you change to this one instead?

https://github.com/enkore/i3pystatus/commit/7a39c3527566411eb1b3e4f79191839ac4b0424e.patch

Also please add the link for the PR that this patch refers too.

Reply:
You may also change to `fetchpatch2` instead.

Reply:
Done